include/bluetooth/services/ots.h

```diff
@@ -468,6 +470,76 @@ struct bt_ots_feat {
 	uint32_t olcp;
 } __packed;
 
+/** @brief Object metadata request bit field values */
+enum {
```

Comment:
Consider adding a name, and use `enum <name>` in the argument where it's used (not necessary, as we have plenty of APIs that don't do this).

Reply:
Prefer not to - as this is a bitfield, combinations of the values are also valid.
Perhaps it would have been better to find another way of documenting these - it is an enum just to be able to attach a Doxygen comment to it.

Reply:
Hmm, I see. 

We have used (unnamed) enums for bitfields before  (e.g. in bluetooth.h), so that's not really an issue, but you just don't prefer to refer to this enum in the function arguments? I don't have a strong opinion here, but personally I think it's fine to use a named enum for bitfields.

Reply:
IMO, using an enum as the type for the argument communicates "use one of these values".  While in fact a lot of other values (combinations of the values) are also valid.
They values are referred to in the documentation for the function, though.

Alternatively, I could add a Doxygen group (I think it was you suggested).